index.js

```diff
@@ -83,7 +57,40 @@ let verifyJwt = function(jwtToken, publicKey, aud, iss) {
             return false;
         }
     },
+
     // check if the kid has a public key 
     hasPublicKey = function(jwtKid) {
         return publicKeys.hasOwnProperty(jwtKid);
-    }
\ No newline at end of file
+    };
+
+// verify the jwtToken against the given configuration
+exports.verify = function(jwtToken, config) {
```

Comment:
Didn't modify it as it would break the apps which are dependent on this library. @chamathsilva this is a good suggestion for the next release.